UPGRADE-FROM-1.12-WITH-PRICE-HISTORY-PLUGIN-TO-1.13.md

```diff
@@ -0,0 +1,42 @@
+⚙️ Upgrade from Sylius 1.12 with PriceHistoryPlugin to Sylius 1.13
```

Comment:
IMO it should be a part of UPGRADE-1.13 file, at most, the legacy instruction can be separated out

Reply:
I'd be for leaving those files as they are and simply adding a note to the main upgrade file, something like `If you've been using the Sylius\PriceHistoryPlugin follow the instructions in <link>`.
It's an optional plugin, leaving the instructions in a separate file will reduce the clutter a bit.